new.js

```diff
@@ -55,3 +55,38 @@ document.getElementById('add-book-form').addEventListener('submit', (e) => {
 
 // Method to del book
 Awesomebooks.delBook();
+
+// Add date
+document.addEventListener('DOMContentLoaded', () => {
+  const today = new Date();
+  const date = `${today.getFullYear()}-${today.getMonth() + 1}-${today.getDate()}`;
+  const time = `${today.getHours()}:${today.getMinutes()}:${today.getSeconds()}`;
+  const dateTime = `${date} ${time}`;
+  document.querySelector('.date').innerHTML = dateTime;
+});
+
+const link = document.getElementById('link-a');
+const addNew = document.getElementById('add-new-a');
+const contact = document.getElementById('contact-a');
+
+const sectionBook = document.querySelector('.book-list-section');
+const addNewBook = document.getElementById('add-book-form');
+const sectionContact = document.querySelector('.contact-section');
+
+link.addEventListener('click', () => {
+  addNewBook.style.display = 'none';
+  sectionContact.style.display = 'none';
+  sectionBook.style.display = 'block';
+});
+
+addNew.addEventListener('click', () => {
+  sectionBook.style.display = 'none';
+  sectionContact.style.display = 'none';
+  addNewBook.style.display = 'flex';
+});
+
+contact.addEventListener('click', () => {
+  sectionBook.style.display = 'none';
+  addNewBook.style.display = 'none';
+  sectionContact.style.display = 'block';
+});
```

Comment:
- [ ] I cannot see the different sections appear upon clicking on any of the links in the navigation bar. For this milestone, you're required to add the navigation bar and when any of the links is clicked, the content for that section only appears on the screen. For instance, when you click the Add New link, the form to add books should appear and when you click the List link, the list of books stored in Local storage or books already added are shown on the screen. Kindly fix that 👍
- [ ] At the moment I cannot give feedback on your project if everything works fine as it should when nothing is being rendered to the screen.

Reply:
When we run the project on all our browsers the links work fine. We have just added id links to the links to see if any change will happen